app/assets/javascripts/procedures.js.coffee

```diff
@@ -38,4 +38,13 @@ $(document).on 'change', 'input.toggle_credit_procedure', ->
     data:
       procedure:
         credited: credited
-        invoiced: !credited
\ No newline at end of file
+        invoiced: !credited
+
+$(document).on 'click', '.procedure_move_button',  ->
```

Comment:
Would it be possible to replace these buttons with anchors instead of using the added javascript?

Reply:
I would prefer to keep it like this, since it matches the existing code structure for other ajax requests throughout the appointments logic.